TEI/ParlaMint-schemaSpecs.editing.odd.xml

```diff
@@ -1,1703 +1,4094 @@
+        early in the morning of <date>10 Dec 1960</date>
+            </birth>
+         </egXML>
+      </exemplum>
+   </elementSpec>
+   <elementSpec ident="death" module="namesdates" mode="change">
+      <desc xmlns:xi="http://www.w3.org/2001/XInclude"
+            versionDate="2005-12-13"
+            xml:lang="en">contains information about a person's death, such as its date and place.</desc>
+      <content>
+         <alternate>
+            <elementRef key="placeName" minOccurs="0" maxOccurs="1"/>
+            <textNode/>
+         </alternate>
+      </content>
+      <attList>
+         <attDef ident="when" mode="change" usage="req"/>
+      </attList>
+      <exemplum xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">
+         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
+            <death when="1960-12-10">Passed away near <name type="place">Aix-la-Chapelle</name>, after suffering from cerebral palsy. </death>
+         </egXML>
+      </exemplum>
+   </elementSpec>
+   <elementSpec ident="placeName" module="namesdates" mode="change">
+      <desc xmlns:xi="http://www.w3.org/2001/XInclude"
+            versionDate="2005-01-14"
+            xml:lang="en">contains an absolute or relative place name.</desc>
+      <content>
+         <alternate>
+            <elementRef key="name" minOccurs="0" maxOccurs="1"/>
+            <textNode/>
+         </alternate>
+      </content>
+      <attList>
+         <pm:attPreserve ident="ref"/>
+      </attList>
+      <exemplum xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">
+         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
+            <placeName>
+               <settlement>Rochester</settlement>
+               <region>New York</region>
+            </placeName>
+         </egXML>
+      </exemplum>
+      <exemplum xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">
+         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
+            <placeName>
+               <geogName>Arrochar Alps</geogName>
+               <region>Argylshire</region>
+            </placeName>
+         </egXML>
+      </exemplum>
+      <exemplum xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">
+         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
+            <placeName>
+               <measure>10 miles</measure>
+               <offset>Northeast of</offset>
+               <settlement>Attica</settlement>
+            </placeName>
+         </egXML>
+      </exemplum>
+   </elementSpec>
+   <elementSpec ident="affiliation" module="namesdates" mode="change">
+      <desc xmlns:xi="http://www.w3.org/2001/XInclude"
+            versionDate="2005-01-14"
+            xml:lang="en">contains an informal description of a person's present or past affiliation with some
+    organization, for example an employer or sponsor.</desc>
+      <content>
+         <textNode/>
```

Comment:
```suggestion
```

Reply:
This change has two problems, so I won't commit it:
* it belongs to the documentation branch rather than to documentation-desc, which is meant only for descriptions of elements
* it breaks current BG data, which often (always?) uses text content in affiliation.